daemon/src/common/FileSystem.cpp

```diff
@@ -122,6 +122,13 @@ template<> struct SerializeTraits<FS::LoadedPakInfo> {
 
 namespace FS {
 
+static Cvar::Cvar<bool> fs_legacypaks("fs_legacypaks", "Don't use versioned pk3 files", Cvar::NONE, false);
```

Comment:
On second thought, we can leave this as is. All the other filesystem cvars are stil fs_* so changing this one a lone would be confusing.